plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/palette/PaletteManagerImpl.java

```diff
@@ -399,12 +399,9 @@ private void replaceNoteAttachmentCreationToolIfNeeded() {
 
     private CreationToolEntry getNoteAttachementToolEntry(final PaletteContainer container) {
         String noteAttachmentToolLabel = Platform.getResourceString(org.eclipse.gmf.runtime.diagram.ui.internal.DiagramUIPlugin.getInstance().getBundle(), "%NoteAttachmentTool.Label"); //$NON-NLS-1$
-        for (Object child : container.getChildren()) {
-            if (child instanceof CreationToolEntry) {
-                CreationToolEntry paletteToolEntry = (CreationToolEntry) child;
-                if (noteAttachmentToolLabel.equals(paletteToolEntry.getLabel())) {
+        for (var child : container.getChildren()) {
```

Comment:
OK this one has also been updated for GEF 3.20 in https://github.com/eclipse/gef-classic/pull/365. We should use `PaletteEntry` instead of var:
```java
       for (PaletteEntry child : (List<? extends PaletteEntry>) container.getChildren()) {
```